src/core/document.js

```diff
@@ -255,7 +254,7 @@ var Page = (function PageClosure() {
         for (i = 0, ii = annotations.length; i < ii; i++) {
           if (isAnnotationRenderable(annotations[i], intent)) {
             opListPromises.push(annotations[i].getOperatorList(
-              partialEvaluator, task, renderInteractiveForms));
+              partialEvaluator, task, true));
```

Comment:
Revert this change. I presume it was for testing your change, but we do want to use the preference here. Tip: when you're developing, open the console and run `PDFViewerApplication.preferences.set('renderInteractiveForms', true);` (and refresh the page) to enable this preference locally so you can easily test it.

Reply:
This is 100% true, thank you!